temba/locations/migrations/0001_initial.py

```diff
@@ -18,12 +18,18 @@ class Migration(migrations.Migration):
         migrations.CreateModel(
             name='AdminBoundary',
             fields=[
-                ('id', models.AutoField(verbose_name='ID', serialize=False, auto_created=True, primary_key=True)),
-                ('osm_id', models.CharField(help_text=b'This is the OSM id for this administrative boundary', unique=True, max_length=15)),
-                ('name', models.CharField(help_text=b'The name of our administrative boundary', max_length=128)),
-                ('level', models.IntegerField(help_text=b'The level of the boundary, 0 for country, 1 for state, 2 for district')),
-                ('geometry', django.contrib.gis.db.models.fields.MultiPolygonField(help_text=b'The full geometry of this administrative boundary', srid=4326, null=True)),
-                ('simplified_geometry', django.contrib.gis.db.models.fields.MultiPolygonField(help_text=b'The simplified geometry of this administrative boundary', srid=4326, null=True)),
+                ('id', models.AutoField(verbose_name='ID',
+                                        serialize=False, auto_created=True, primary_key=True)),
+                ('osm_id', models.CharField(
+                    help_text=b'This is the OSM id for this administrative boundary', unique=True, max_length=15)),
+                ('name', models.CharField(
+                    help_text=b'The name of our administrative boundary', max_length=128)),
+                ('level', models.IntegerField(
+                    help_text=b'The level of the boundary, 0 for country, 1 for state, 2 for district')),
+                ('geometry', django.contrib.gis.db.models.fields.MultiPolygonField(
+                    help_text=b'The full geometry of this administrative boundary', srid=4326, null=True)),
+                ('simplified_geometry', django.contrib.gis.db.models.fields.MultiPolygonField(
+                    help_text=b'The simplified geometry of this administrative boundary', srid=4326, null=True)),
```

Comment:
You cannot change PAST migrations, that's the whole point of migrations. You need to create new migrations to evolve the old models to the new ones and write data migrations to alter existing data into the new format.